packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -222,3 +222,23 @@ def assert_packages_match_active_buildinfo(package_ids):
 
         assert set(active_package_ids) <= set(package_ids)
         assert_packages_match_active_buildinfo(active_package_ids)
+
+
+def test_packaging_api(dcos_api_session):
+    """Test the Cosmos API (/package) wrapper
+    """
+    install_response = dcos_api_session.cosmos.install_package('kafka', '1.1.9-0.10.0.0')
```

Comment:
Ah good point. It worked when I manually ran the current version of this PR, so it seems like 2 is fine but I'll check.

Would it be a problem increasing it to the required 5 agents for the tweeter integration test? Seems kind of wasteful if there's only the one integration test that uses it